src/components/templates/Home.tsx

```diff
@@ -13,7 +13,7 @@ export function Home() {
     <>
       <LandingPageLinks pages={[...recentWorkPageInfo.orderedPages]} />
       <VideoList videos={videos} pageType="recent" />
-      <Contact />
+      <Contact id="contact" />
```

Comment:
❓ Does this need to be a prop?  Since the component is called `Contact`, would it ever be a different value?

Reply:
The only reason I can see this wanting to be a prop is if we ever render two contact components. But I don't think we do, so I think it makes sense to keep it inside the component.